telingo/theory/formula.py

```diff
@@ -88,6 +88,29 @@ def false_literal(self):
 """
 g_tel_operators = {"<", ">", "<:", ">:", "<*", ">*", ">?", "<?", ">>", "<<", "<;", "<:;", ";>", ";>:"}
 
+"""
+Set of dynamic connectives.
+"""
+g_del_operators = {".>*", ".>?"}
+
+"""
+Set of unary Path operators.
+
+"""
+g_path_unary_operators  = {"*", "?"}
+
+"""
+Set of binary Path operators.
+
+"""
+g_path_binary_operators = {";;", "+"}
+
+"""
+Set of all tel, del and path operators.
+
+"""
+g_all_operators = g_binary_operators.union(g_unary_operators).union(g_arithmetic_operators).union(g_tel_operators).union(g_del_operators).union(g_path_unary_operators).union(g_path_binary_operators)
```

Comment:
The `set.union` takes an arbitrary number of sets as argument. With this you can make it look less horrible.